src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js

```diff
@@ -286,6 +286,10 @@ function createRow(data, row)
 
     if (!data["doi"]["doi"]) {
         $(rowContent).find("#container-dataset-doi").hide();
+    } else if (activeTabIndex == 2) {
+        // Remove the hyperlink, but keep text for this tab.
+        $(rowContent).find("#dataset-doi").attr("href", $(rowContent).find("#dataset-doi").attr("href")).removeAttr("href");
```

Comment:
Why not simply removeAttr? Or you can use replaceWith.
(And you won't need line 292).